### ISSUE TITLE
[Feat]: Add labels to cgroups (containers and vms) and docker charts

### ISSUE THREAD

**Issue by shyamvalsan** [netdata/netdata#14799]
### Problem

The container, vm and docker charts currently have no labels and the instance value (which may or may not be easy to understand or remember) is the only thing they have to identify what's going on. 

### Description

Please add the following labels to docker charts: 

- container name
- container image
- container tag

On a user interview call, a user raised the fact that they have hundreds of ephemeral containers and the container id string it is hard for them to make sense of all the data which just the container instance value.  

Another user has highlighted this as a problem in their troubleshooting journey.

### Importance

must have

### Value proposition

1. Make container and docker monitoring more usable

**Comment by ilyam8** [comment]
> on the other hand when I tried to get image and tags it was not possible
> Am I missing something?

Hard to say, because you did explain what you did. Can you explain "I tried to get image and tags" part, please?

**Comment by thiagoftsm** [comment]
Hello @ilyam8 ,

Reading official documentation in docker site, I observed that some information is stored in `/var/lib/docker/`. And the specific information that were required in this issue is available in my environment in the following file:

```sh
root@hades:/home/thiago/Netdata/netdata# ls -l /var/lib/docker/containers/1977e27040aa36cbf83378faeebb4dadcb38ef51b3d01760876cddf31f0a932c/config.v2.json 
-rw------- 1 root root 2553 Apr  4 12:00 /var/lib/docker/containers/1977e27040aa36cbf83378faeebb4dadcb38ef51b3d01760876cddf31f0a932c/config.v2.json
```

that netdata cannot read.

**Comment by ilyam8** [comment]
tl;dr We need to check if we support labels with no value because not all cgroups will have the "image" label.

--

@shyamvalsan, hey. We haven't added "image" because I am not sure how we deal with labels that don't belong to all cgroups.

I installed [cAdvisor](https://github.com/google/cadvisor) and configured Prometheus to scrape from it. I see that cAdvisor adds `image=""` for all cgroups that have no image. I think there is a requirement - in order to do operations with metrics they need to have the same label set.

See the 3rd metric, it is all cgroups with no image.

<img width="1725" alt="Screenshot 2023-04-07 at 14 30 29" src="https://user-images.githubusercontent.com/22274335/230601718-105551cf-9784-4ab7-955e-1c55578370ae.png">

**Comment by shyamvalsan** [comment]
Why have labels only been added to cgroup charts and not to the docker charts? Shouldn't we have the same labels for these as well? @ilyam8 @thiagoftsm 

![image](https://user-images.githubusercontent.com/24860547/230909282-046ac262-fcc6-441a-9af4-e10b65ca0571.png)

**Comment by ilyam8** [comment]
@shyamvalsan, hey. That is not possible because the chart you are showing is a summary chart (not a docker container chart).

**Comment by ilyam8** [comment]
> container image
> container tag

@shyamvalsan The image label contains both the image name and tag.

**Comment by ilyam8** [comment]
We can't group them by labels because we don't collect per-container metrics.

**Comment by shyamvalsan** [comment]
Because? Are those metrics just not available per container? I should be able to tell what size which container is on Netdata imo.

**Comment by shyamvalsan** [comment]
I was talking about this from [ContainerList](https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerList). 

![image](https://user-images.githubusercontent.com/24860547/231080375-a8b458e7-5cd2-41c5-8fb6-c6bdd19ab69b.png)

There also seem to be filters for health in ContainerList.

**Comment by ilyam8** [comment]
This is a filter, the response (Response Sample) doesn't contain health status.

**Comment by shyamvalsan** [comment]
@ilyam8 ok, my thought was we could query for containers by different health status fields separately. Or do you mean this would be too intensive a query as well? 

Also does the size parameter I highlighted above, answer the previous question?
